cloud-optimized-geotiffs/writing-cogs-in-python.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>We should probably paraphrase, include a short version of the best practices as to not rely on an external source.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p><em>Thanks! I have added a few recommendations in the notebook and added the link to additional resources.</em></p>

Reply:
<pre class="ql-syntax" spellcheck="false">Some of the recommendations while generating COGs are noted below -

1. Using&nbsp;WEBP&nbsp;compression for RGB or RGBA dataset (there is a lossless option). Although, this is considered as a best option if you are looking for space saving, JPEG compression might be considered a safer choice.
2. Using&nbsp;Deflate&nbsp;compression with&nbsp;PREDICTOR=2&nbsp;and&nbsp;ZLEVEL=9&nbsp;options for non-Byte or non RGB datasets. The&nbsp;PREDICTOR&nbsp;value is set for&nbsp;LZW,&nbsp;DEFLATE&nbsp;and&nbsp;ZSTD&nbsp;compression.&nbsp;PREDICTOR=2&nbsp;or standard predictor is used for integer data type and&nbsp;PREDICTOR=3&nbsp;or floating-point predictor is used for floating point data type.
3. Using internal overviews.
4. Using a internal block size of&nbsp;256&nbsp;or&nbsp;512
</pre>